src/commands/fun.rs

```diff
@@ -0,0 +1,31 @@
+use crate::try_send_message_content;
+use rand::seq::SliceRandom;
+use rand::thread_rng;
+use serenity::{
+    framework::standard::{
+        macros::{command, group},
+        Args, CommandError, CommandResult,
+    },
+    model::prelude::*,
+    prelude::*,
+};
+
+group!({
+    name: "fun",
+    options: {},
+    commands: [choose]
+});
+
+#[command]
+pub fn choose(ctx: &mut Context, msg: &Message, args: Args) -> CommandResult {
+    let options: Vec<&str> = args.rest().split(", ").collect();
```

Comment:
If we upgrade to `serenity = "0.6.4"`, we'll be able to use per-command delimiters, potentially allowing us to set the delimiter as `", "` and then do `args.iter().collect();`